sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3777,6 +3777,23 @@ object functions {
     window(timeColumn, windowDuration, windowDuration, "0 second")
   }
 
+  /**
+   * Extracts the event time from the window column of a record produced by window aggregation
```

Comment:
I may need to be strict here to avoid further confusion based on comment. We do not have window aggregation operator. Technically saying, window function is effectively a TVF like explode(), and we apply the "normal" aggregation against the grouped Dataset which group contains window.

That said, `produced by window aggregation` seems to be unnecessary.

Reply:
reworded